frontend/src/v5/ui/components/shared/revisionDetails/components/revisionsListItemButton/revisionsListItemButton.component.tsx

```diff
@@ -26,12 +28,24 @@ type IRevisionsListItemButton= {
 
 export const RevisionsListItemButton = ({ status, onClick }: IRevisionsListItemButton): JSX.Element => {
 	const isVoid = !!status;
-	const textStatus = isVoid ? 'void' : 'active';
-	const changeToStatus = isVoid ? 'active' : 'void';
+	const textStatus = isVoid
+		? formatMessage({ id: 'revisionDetails.list.item.button.void', defaultMessage: 'void' })
+		: formatMessage({ id: 'revisionDetails.list.item.button.active', defaultMessage: 'active' });
+	const changeToStatus = isVoid
+		? formatMessage({ id: 'revisionDetails.list.item.button.tooltip.active', defaultMessage: 'active' })
+		: formatMessage({ id: 'revisionDetails.list.item.button.tooltip.void', defaultMessage: 'void' });
```

Comment:
I think we could use make it less wordy. 

What do you think about this?
```suggestion
       const voidStr =  formatMessage({ id: 'revisionDetails.void', defaultMessage: 'void' });
       const activeStr =  formatMessage({ id: 'revisionDetails.active', defaultMessage: 'active' });

	const textStatus = isVoid ? voidStr : activeStr;
	const changeToStatus = isVoid ? activeStr : voidStr;
```